### PR TITLE
Merging front-end to main: Custom order list, UI Improvements

### PR DESCRIPTION
**Pull request by slok1368**

### CODE REVIEW

File: src/front-end/front-end-interface/AddGameScreen.cs [hunk 1]
```diff
@@ -6,25 +6,31 @@
...
         [DllImport("LinkFrontAndBack.dll")]
         public static extern void add([In] string game_name, [In] string game_path, [In] string image_path);
-        public AddGameScreen(RefreshListFunc refreshListFunc)
+        public AddGameScreen()
         {
             InitializeComponent();
-            this.refreshListFunc = refreshListFunc;
+        }
+
+        public AddGameScreen(MainScreen main)
```
Comment (asagrekin) [inline src/front-end/front-end-interface/AddGameScreen.cs:24]: Can we get some comments on the public functions?

File: src/front-end/front-end-interface/AddGameScreen.cs [hunk 2]
```diff
@@ -36,14 +42,16 @@ private void ConfirmAddGameButton_Click(object sender, EventArgs e)
                 imagePath.EndsWith(".jpg") ||
                 imagePath.EndsWith(".jpeg")))
             {
-                imagePath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) + "\\src\\front-end\\front-end-interface\\Resources\\Logo.png";
+                imagePath = "Resources/Logo.png";
```
Comment (asagrekin) [inline src/front-end/front-end-interface/AddGameScreen.cs:45]: "/" vs "\" for windows? I see it was used interchangeably?

File: src/front-end/front-end-interface/MainScreen.cs [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
+using System;
 using System.Collections.Generic;
+using System.Drawing;
+using System.Net.NetworkInformation;
 using System.Runtime.InteropServices;
 using System.Text;
 using System.Windows.Forms;
 
-namespace Games_In_One_App
+namespace Games_In_One
```
Comment (asagrekin) [inline src/front-end/front-end-interface/MainScreen.cs:10]: Good job shortening names!

File: src/front-end/front-end-interface/MainScreen.cs [hunk 4]
```diff
@@ -15,17 +18,64 @@ public partial class MainScreen : Form
...
+            // Games Scroll Panel
+            this.gamesScrollPanel = CreateScrollPanel(this.gamesList);
+            this.Controls.Add(this.gamesScrollPanel);
+
+            // Scroll Panel
+            this.editScrollPanel = CreateScrollPanel(this.gamesEditList);
+            this.Controls.Add(this.editScrollPanel);
+
+        }
+        private Panel CreateScrollPanel(TableLayoutPanel tablePanel)
```
Comment (Kelbaso) [inline src/front-end/front-end-interface/MainScreen.cs:67]: A great addition to user usability, nice thinking Steven!

**Review by asagrekin — APPROVED** [review]
Looks really good, and I like the layout! In the future, please comment public functions.

File: src/front-end/front-end-interface/AddGameScreen.cs (hunk 2 above)
Comment (Kelbaso) [inline src/front-end/front-end-interface/AddGameScreen.cs:45]: Great simplification of the original imagePath collection.

File: src/front-end/front-end-interface/GameRow.cs [hunk 5]
```diff
@@ -72,5 +91,16 @@ protected override void OnPaint(PaintEventArgs e)
                 e.Graphics.DrawLine(pen, 0, Height - 1, Width, Height - 1);
             }
         }
+
+        public void removeFromDB()
```
Comment (lochoa2) [inline src/front-end/front-end-interface/GameRow.cs:99]: these 2 methods below may need some comments.

File: src/front-end/front-end-interface/Program.cs [hunk 6]
```diff
@@ -16,7 +17,24 @@ static void Main()
...
-            Application.Run(new MainScreen());
+            MainScreen mainScreen = new MainScreen();
+            string pngFilePath = "Resources/Logo.png";
+            string iconFilePath = "Resources/Logo.ico";
+            ConvertPngToIcon(pngFilePath, iconFilePath);
+            mainScreen.Icon = new Icon(iconFilePath);
+            Application.Run(mainScreen);
+        }
+
+        static void ConvertPngToIcon(string pngFilePath, string iconFilePath)
```
Comment (Kelbaso) [inline src/front-end/front-end-interface/Program.cs:28]: Another useful addition to the quality of our product.

**Review by lochoa2 — COMMENTED** [review]
review complete

**Review by Kelbaso — APPROVED** [review]
Finished Reviewing The Code, Great Additions Steven!

File: src/front-end/front-end-interface/AddGameScreen.cs (hunk 1 above)
Comment (slok1368) [reply src/front-end/front-end-interface/AddGameScreen.cs:24]: Added comments to public functions.

File: src/front-end/front-end-interface/AddGameScreen.cs (hunk 2 above)
Comment (slok1368) [reply src/front-end/front-end-interface/AddGameScreen.cs:45]: Yes, '/' slash works too.

File: src/front-end/front-end-interface/GameRow.cs (hunk 5 above)
Comment (slok1368) [reply src/front-end/front-end-interface/GameRow.cs:99]: Added comments

**Pull request merged**